starwars/src/App.js

```diff
@@ -1,19 +1,41 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
+import axios from "axios";
+import StarWarsCard from "./components/StarWarsCard"
+import "./components/StarWars.css"
 
 const App = () => {
```

Comment:
Keep in mind that normally we just want App.js to have as little info and logic as possible. I'd recommend creating a list/container component and doing your axios call and setting state there next time. The way you set up your card component looks awesome though!